docs/tutorials/uploading-photo-to-a-product.md

```diff
@@ -0,0 +1,119 @@
+# Uploading Images to the Open Food Facts API
+<!--Add a brief introduction of what the tutorial does -->
+
+This basic tutorial shows you can upload image of a product to the Open Food Facts API.
+
+## Points to consider before uploading photos
+
+### Image license
+
+Products images MUST be under the [Creative Commons Attribution ShareAlike licence](https://creativecommons.org/licenses/by-sa/3.0/deed.en)(https://creativecommons.org/licenses/by-sa/3.0/deed.en).
+
+That means you should either upload: 
+* photos that are your own work
+* photos taken by your users, with their consent for this license (should be part of your service terms)
+* photos already under this license or a more liberal one
+
+### Image Quality
+
+Uploading quality photos of a product, its ingredients and nutrition table is very important, since it allows the Open Food Facts OCR system to retrieve important data to analyze the product. The minimal allowed size for photos is 640 x 160 px.
+
+### Upload Behavior
+
+In case you upload more than one photo of the front, the ingredients and the nutrition facts, beware that only the first photo of each category will be displayed. (You might want to take additional images of labels, recycling instructions, and so on). However, all photos will be saved.
```

Comment:
```suggestion
In case you upload more than one photo of the front, the ingredients, the nutrition facts, or the product packaging components, beware that only the latest "selected" photo of each category will be displayed on the product page on the website and on the mobile application. 

The older ones are still saved, and can be "selected" either by an API call or via the editing interface (website and mobile application).

You can also upload some photos that are neither of that 4 categories, but they will not be displayed by default.. However, all photos will be saved.
```